### PR TITLE
build: update golang to 1.13

### PR DESCRIPTION
**Pull request by RuriRyan**
Update golang to the latest stable release version.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -113,7 +113,7 @@ RUN chmod +x /usr/local/bin/composer
 
 RUN apt-get update && apt-get install -y bzr && apt-get clean
 
-ENV GOLANG_VERSION 1.12
+ENV GOLANG_VERSION 1.13
```
Comment (ikesyo) [inline Dockerfile]: Let's use the latest `1.13.2`

```suggestion
ENV GOLANG_VERSION 1.13.2
```

**Review by ikesyo — COMMENTED** [review]
I think we should do the same thing as https://github.com/renovatebot/docker-go/pull/16.

File: Dockerfile (hunk 1 above)
Comment (piepmatz) [reply Dockerfile]: @ikesyo, the `1.13` label automatically refers to the latest patch version. If you explicitly use the patch version here, this line will need to be changed quite often. Is the pinning worth it?

**Comment by rarkins** [comment]
I think we also need to remove `ENV GO111MODULE=on` and add `ENV GOPROXY=direct GOSUMDB=off`

**Comment by piepmatz** [comment]
@rarkins, regarding `GO111MODULE ` I guess you are referring to https://github.com/renovatebot/docker-go/pull/14?
There's no `GO111MODULE` in here :)

File: Dockerfile (hunk 1 above)
Comment (ikesyo) [reply Dockerfile]: > the 1.13 label automatically refers to the latest patch version.

Oh I didn't know that.

**Pull request merged**